apidocs/openapi/certs.yaml

```diff
@@ -81,16 +81,18 @@ paths:
           description: Database can't process request.
         "500":
           $ref: "#/components/responses/ServiceError"
+
+  /{domainID}/certs/{clientID}:
     delete:
       operationId: revokeCert
       summary: Revokes a certificate
       description: |
-        Revokes a certificate for a given cert ID.
+        Revokes a certificate for a given client ID.
       tags:
         - certs
       parameters:
         - $ref: "auth.yaml#/components/parameters/DomainID"
-        - $ref: "#/components/parameters/CertID"
+        - $ref: "#/components/parameters/ClientID"
```

Comment:
Also the certID is not a uuid, from the code it is the serial number of the cert, please update the definition of the certID parameter on line 169